### PR TITLE
Move docker-compose logging statement to debug

### PR DESCRIPTION
**Pull request by jasontedor**
When docker-compose is required for a test fixture but is not available, we warn log a message to this effect. This ends up being noise during configuration, especially when working locally. This commit changes the logging level of these messages to debug.

### CODE REVIEW

File: buildSrc/src/main/java/org/elasticsearch/gradle/testfixtures/TestFixturesPlugin.java [hunk 1]
```diff
@@ -136,7 +136,7 @@ public void apply(Project project) {
 
         if (dockerComposeSupported() == false) {
             project.getLogger()
-                .warn(
+                .debug(
```
Comment (mark-vieira) [inline buildSrc/src/main/java/org/elasticsearch/gradle/testfixtures/TestFixturesPlugin.java]: Can we use `info` instead? Debug level logging is verbose to the point of being nearly useless and `info` is not logged by default either unless you ask with `--info`.

**Review by mark-vieira — APPROVED** [review]
I'd prefer info vs debug, otherwise LGTM.

File: buildSrc/src/main/java/org/elasticsearch/gradle/testfixtures/TestFixturesPlugin.java (hunk 1 above)
Comment (jasontedor) [inline buildSrc/src/main/java/org/elasticsearch/gradle/testfixtures/TestFixturesPlugin.java]:
```suggestion
                .info(
```

Comment (jasontedor) [reply buildSrc/src/main/java/org/elasticsearch/gradle/testfixtures/TestFixturesPlugin.java]: Done.

**Comment by jasontedor** [comment]
@elasticmachine update branch

**Review by mark-vieira — APPROVED** [review]
👍

**Pull request merged**